src/api/store/community.py

```diff
@@ -1381,7 +1382,7 @@ def list_communities_feature_flags(self, context, args) -> Tuple[list, MassEnerg
                 if not community:
                     return None, CustomMassenergizeError("Community not found")
                 
-                communities = [community.id]
+                return get_enabled_flags(community), None
```

Comment:
is there a corresponding FE change for this?  do they need to go in at the same time?